content/pt/docs/concepts/instrumentation/zero-code.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sem código
+description: >-
+  Aprenda como adicionar observabilidade a uma aplicação sem precisar de
+  escrever código
+weight: 10
+default_lang_commit: 702cb43a65464842ebfbf5cb1ac6cf5c72c48462
+---
+
+Como [operações](/docs/getting-started/ops/), você pode querer adicionar
+observabilidade a uma ou mais aplicações sem precisar editar o código-fonte. O
+OpenTelemetry permite que você ganhe rapidamente alguma observabilidade para um
+serviço sem precisar usar a API & SDK do OpenTelemetry para
```

Comment:
Sugestão:
"a service without having to use..." -> "um serviço sem ter que usar..."